docs/generators/typescript-nestjs.md

```diff
@@ -0,0 +1,29 @@
+
+---
+id: generator-opts-client-typescript-nestjs
+title: Config Options for typescript-nestjs
+sidebar_label: typescript-nestjs
+---
+
+| Option | Description | Values | Default |
+| ------ | ----------- | ------ | ------- |
+|sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
+|ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
+|allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
+|prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
+|modelPropertyNaming|Naming convention for the property: 'camelCase', 'PascalCase', 'snake_case' and 'original', which keeps the original name| |camelCase|
+|supportsES6|Generate code that conforms to ES6.| |false|
+|npmName|The name under which you want to publish generated npm package. Required to generate a full package| |null|
+|npmVersion|The version of your npm package. If not provided, using the version from the OpenAPI specification file.| |1.0.0|
+|snapshot|When setting this property to true, the version will be suffixed with -SNAPSHOT.yyyyMMddHHmm| |false|
+|npmRepository|Use this property to set an url your private npmRepo in the package.json| |null|
+|withInterfaces|Setting this property to true will generate interfaces next to the default class implementations.| |false|
+|taggedUnions|Use discriminators to create tagged unions instead of extending interfaces.| |false|
+|providedInRoot|Use this property to provide Injectables in root (it is only valid in nestjs version greater or equal to 6.0.0).| |false|
```

Comment:
could you please clarify how this option is used in the generated code?

Reply:
It is not used and should be removed!